pkg/standalone/runfileconfig/run_file_config.go

```diff
@@ -41,3 +50,57 @@ type EnvItems struct {
 	Name  string `yaml:"name"`
 	Value string `yaml:"value"`
 }
+
+func (a *App) GetLogsDir() string {
+	logsPath := filepath.Join(a.AppDirPath, ".dapr", "logs")
+	if _, err := os.Stat(logsPath); errors.Is(err, os.ErrNotExist) {
+		os.MkdirAll(logsPath, 0o755)
+	}
+	return logsPath
+}
+
+func (a *App) GetAppLogFileWriter() (io.WriteCloser, error) {
+	logsPath := a.GetLogsDir()
+	f, err := os.Create(filepath.Join(logsPath, "app.log"))
```

Comment:
f.close() ? 
and should we use `uuid` suffix in `app.log`

Reply:
done